### PR TITLE
Single Video Added

### PR DESCRIPTION
**Pull request by m26unkwn**
Single video Page added

### CODE REVIEW

**Review by saurabhchirde — APPROVED** [review]
Couldn't find any issue, as your code is clean ad easy to read. 
Good to merge.

File: src/components/Cards/video/index.jsx [hunk 1]
```diff
@@ -6,17 +6,18 @@ import { Link } from "react-router-dom";
 import { ReactComponent as MenuIcon } from "../../../assets/ElipsesIcon.svg";
 
 export const VideoCard = ({ props }) => {
-  const { thumbnail, title, id } = props;
+  const { thumbnail, title, _Id } = props;
+
   return (
     <div className=' video-card card-container flex flex-col '>
```
Comment (manojsarna) [inline src/components/Cards/video/index.jsx:12]: Suggestion : change the filename to VideoCard.jsx

File: src/screens/singlevideo/SingleVideo.jsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+import "./singleVideo.css";
+
+export const SingleVideo = () => {
+  const { videoId } = useParams();
+  const [singleVideo] = useAxios({
+    method: "get",
+    url: `/api/video/${videoId}`,
+    property: "video",
+  });
```
Comment (manojsarna) [inline src/screens/singlevideo/SingleVideo.jsx:18]: Appreciation : good use of the useAxios hook

File: src/screens/singlevideo/singleVideo.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+.video-head {
+  gap: 0.5rem;
+}
+.video-title {
+  color: var(--heading-text-color);
+}
+.video-views {
+  color: #949393;
+}
+.video-action {
```
Comment (manojsarna) [inline src/screens/singlevideo/singleVideo.css:30]: Suggestion : use color variables whenever possible

File: src/screens/singlevideo/singleVideo.css [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  color: var(--heading-text-color);
+}
+
+@media only screen and (max-width: 700px) {
+  .video-player {
+    height: 20rem;
+  }
+}
+@media only screen and (max-width: 400px) {
+  .video-player {
```
Comment (manojsarna) [inline src/screens/singlevideo/singleVideo.css:49]: suggestion : keep standard media queries width sizes
1024px, 768px , 480 px etc

**Review by manojsarna — APPROVED** [review]
Overall the code is good

Do check the comments

File: src/screens/singlevideo/singleVideo.css (hunk 4 above)
Comment (m26unkwn) [reply src/screens/singlevideo/singleVideo.css:49]: Thanks Manoj For suggesting these things

**Review by dev-ashishsethi — APPROVED** [review]
learnt about fragment component from your code thanks! all looks good

**Pull request merged**